### PR TITLE
Updates to the alerts and automation pilot tutorials

### PR DESCRIPTION
**Pull request by danielva**
based on feedback from Witalij, Tom, and Kaloyan

### CODE REVIEW

File: tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ primary_tag: software-product>sap-hana-cloud
...
   - How to trigger an SAP Automation Pilot command from the Alert Notification Service
 
-An SAP HANA Cloud database or a data lake Relational Engine instance have a set of built-in alerts that when triggered, are sent to the SAP Alert Notification Service (ANS).  This service in addition to being able to forward details of the alert to various channels (email, Microsoft Teams, Slack, etc.), can trigger an [SAP Automation Pilot](https://discovery-center.cloud.sap/serviceCatalog/automation-pilot?region=all) command.  In this tutorial, a command will be created to attempt to resolve a disk use alert.
+An SAP HANA Cloud database or a data lake Relational Engine instance have a set of built-in alerts that when triggered, are sent to the SAP Alert Notification Service (ANS).  This service, in addition to being able to forward details of the alert to various channels (email, Microsoft Teams, Slack, etc.), can trigger an [SAP Automation Pilot](https://discovery-center.cloud.sap/serviceCatalog/automation-pilot?region=all) command.  In this tutorial, a command will be created to attempt to resolve a disk use alert.
```
Comment (MichaelCzcz) [inline tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md:20]: A SAP HANA Cloud database...
This service..., can trigger a ... command.

You only use "an" when the following word starts with a vowel

File: tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ primary_tag: software-product>sap-hana-cloud
...
+SAP Automation Pilot includes [provided commands](https://help.sap.com/viewer/de3900c419f5492a8802274c17e07049/Cloud/en-US/5bbe7dba99d24caeafddf7fa62dc63b9.html) for various services in the BTP, a [scheduler](https://help.sap.com/viewer/de3900c419f5492a8802274c17e07049/Cloud/en-US/96863a2380d24ba4bab0145bbd78e411.html), and the ability to create new commands.  Examples of the provided commands for an SAP HANA Cloud, SAP HANA database include start, stop, and restart and for Cloud Foundry include `GetCfServiceInstance` and `UpdateCfServiceInstance`.  Commands can be created using a BASH script, Python, Node.js or Perl as described at [Execute Script](https://help.sap.com/viewer/de3900c419f5492a8802274c17e07049/Cloud/en-US/d0854dbb80d84946bb57791db94b7e20.html).
```
Comment (MichaelCzcz) [inline tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md:24]: Same here.
You only use "an" when the following word starts with a vowel.

File: tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md [hunk 3]
```diff
@@ -55,7 +55,9 @@ The following steps demonstrate how to subscribe to the SAP Automation Pilot ser
 
     ![open application](open-application.png)
 
-4. Notice that it provides a set of catalogues containing provided commands and inputs including ones for Cloud Foundry and database lifecycle management.  Provided catalogs are groups of commands and inputs that are provided by the SAP Automation Pilot, and own catalogues are those that are user created and contain user created commands and inputs.  Catalogues provide a way to group a collection of related commands and inputs.
+    > If you receive a permission error, you may need to log out and log in again for the role assigned in the previous step to take effect.
+
+4. Notice that it provides a set of catalogues containing provided commands and inputs including ones for Cloud Foundry and database lifecycle management.  Provided catalogs are groups of commands and inputs that are provided by the SAP Automation Pilot.  Own catalogues are those that are user created and contain user created commands and inputs.  Catalogues provide a way to group a collection of related commands and inputs.
```
Comment (MichaelCzcz) [inline tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md:60]: Please use either American or British spelling -> catalogs/catalogues.

File: tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md [hunk 4]
```diff
@@ -147,35 +153,35 @@ This step will create a catalog that contains a command and an input.  The input
 
     ![input with keys](input.png)
 
-    >Note these credentials will be used by a Cloud Foundry command to return  the details of an SAP HANA database service.  The user should appear under the members list at the space level in the BTP Cockpit, have the role space developer, and not have two-factor authentication enabled.  Additional details on creating a technical user can be found at [Creating a Technical User for Cloud Platform Integration](https://blogs.sap.com/2018/08/17/creating-a-technical-user-for-cloud-platform-integration/).
+    >Note these credentials will be used by a Cloud Foundry command to return the details of an SAP HANA database service.  Ensure the user can login successfully to a site such as [SAP Community](https://community.sap.com/) and not have two-factor authentication enabled.  The user should appear under the members list at the space level in the BTP Cockpit and have the role space developer.  Additional details on creating a technical user can be found at [Creating a Technical User for Cloud Platform Integration](https://blogs.sap.com/2018/08/17/creating-a-technical-user-for-cloud-platform-integration/).
```
Comment (MichaelCzcz) [inline tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md:156]: the user can log in

to log in, the login

File: tutorials/hana-cloud-alerts/hana-cloud-alerts.md [hunk 5]
```diff
@@ -114,7 +114,7 @@ The following instructions demonstrate a few examples of triggering alerts in an
     END;
     ```
 
-    > Long running statements can be run as a background activity which will continue even if the machine the SAP HANA database explorer is running on loses internet connectivity or the tab is closed.  
+    >Long running statements can be run as a background activities in the SAP database explorer which will continue even if the browser's connection drops or the tab is closed.
```
Comment (MichaelCzcz) [inline tutorials/hana-cloud-alerts/hana-cloud-alerts.md:117]: "run as a background activities in the SAP database"
Please remove "a" or change "activities" to singular.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md (hunk 4 above)
Comment (danielva) [reply tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md]: Will update "Ensure the user can login "
to 
"Ensure the user can log in"

File: tutorials/hana-cloud-alerts/hana-cloud-alerts.md (hunk 5 above)
Comment (danielva) [reply tutorials/hana-cloud-alerts/hana-cloud-alerts.md]: Fixed

File: tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md (hunk 1 above)
Comment (danielva) [reply tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md]: Will make those changes throughout.

File: tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md (hunk 2 above)
Comment (danielva) [reply tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md]: Will make those changes throughout.

**Pull request merged**